code/datums/achievements/boss_achievements.dm

```diff
@@ -103,11 +103,11 @@
 	database_id = BOSS_MEDAL_LEGION_CRUSHER
 	icon_state = "legion"
 
-/datum/award/achievement/boss/wendigo_crusher
-	name = "Wendigo Crusher"
+/datum/award/achievement/boss/hornvandr_crusher
+	name = "Hornvandr Crusher"
 	desc = "You've now ruined years of mythical storytelling."
-	database_id = BOSS_MEDAL_WENDIGO_CRUSHER
-	icon_state = "wendigo"
+	database_id = BOSS_MEDAL_HORNVANDR_CRUSHER
```

Comment:
Will this currently migrate all wendigo achievements to the new one?

Reply:
i dont know

Reply:
Don't touch these if you don't know, they're internal vars and don't need to be changed

Reply:
they will not be migrated automatically so you will need to keep the achievement the same.